src/main.rs

```diff
@@ -134,6 +134,7 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .parse()
                 .unwrap_or(0),
             unsafe_: matches.is_present("unsafe"),
+            escape: matches.is_present("escape"),
```

Comment:
We'll need to add a corresponding `clap::Arg` above to get this option from the command line.

Reply:
I've updated the commit to include command-line support.